test/functional/plugin/lsp_spec.lua

```diff
@@ -1,5 +1,6 @@
 local helpers = require('test.functional.helpers')(after_each)
 
+local vim = vim
```

Comment:
Good catch!

Reply:
The environment of test and the environment of exec_lua are different, so I don't think it's necessary in this case.

Reply:
TIL about 'make lint', fixed.